repos/system_upgrade/el8toel9/actors/checkcustomnetworkscripts/libraries/customnetworkscripts.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from leapp import reporting
+
+CUSTOM_NETWORK_SCRIPTS = [
+    "/sbin/ifup-local",
+    "/sbin/ifup-pre-local",
+    "/sbin/ifdown-local",
+    "/sbin/ifdown-pre-local",
+]
+
+
+def generate_report():
+    """ Generate reports informing user about possible manual intervention requirements """
+
+    DOC_URL = "https://red.ht/upgrading-RHEL-8-to-RHEL-9-network-scripts"
+
+    # Show documentation url if custom network-scripts detected
+    title = "custom network-scripts detected"
+    summary = "RHEL 9 does not support the legacy network-scripts package that was \
```

Comment:
Nit: That's super minor, but could you please use string concatenation instead of \?
Like
```
summary = ("RHEL 9 does not support the legacy network-scripts package that was"
           "deprecated in RHEL 8, and the custom network-scripts have been detected. In order to"
           "proceed with the upgrade, you can move the custom network-scripts, for example, to"
           "the `/opt/network-scripts/` directory."
)
```

Reply:
i missed that. thanks for the socnd eyes @fernflower :)

Reply:
I changed it into string concatenation.